.github/workflows/nodejs.yml

```diff
@@ -7,6 +7,7 @@ on:
     - master
     - develop
     - next
+    - supplier/*
```

Comment:
**Remove this before merge**

Reply:
Mình phải làm gì, ai chỉ mình với , mình người mới ko biết gì